horcrux/src/shamir.rs

```diff
@@ -110,6 +113,10 @@ impl<F: Field + Debug + Display> Shamir<F> for CompactShamir {
     type X = u8;
     type Share = CompactShare<F>;
 
+    fn share(x: Self::X, y: F) -> Self::Share {
```

Comment:
This implementation can directly be given as a default implementation in the trait itself.